### PR TITLE
Update nickname.ts

### PR DESCRIPTION
**Pull request by alexkotr1**

### CODE REVIEW

File: src/discord/commands/moderation/nickname.ts [hunk 1]
```diff
@@ -1,41 +1,36 @@
...
-
-
-        if (!member) {
-            throw new CommandError({
-                message: "Please specify a member of this guild."
-            });
+    handler: async (message) => {
+        const [member,nickname] : GuildMember | any = message.args
+        try {
+            await member.setNickname(nickname ? nickname : message.author.username,'Executed by ' + `${message.author.username}(${message.author.id})`)
```
Comment (nullpixel) [inline src/discord/commands/moderation/nickname.ts]: Please use \`{}\` string interop here

**Review by nullpixel — CHANGES_REQUESTED** [review]
Overall PR is fine, just address my comment. Also might want to run prettier for code style things.

File: src/discord/commands/moderation/nickname.ts (hunk 1 above)
Comment (alexkotr1) [reply src/discord/commands/moderation/nickname.ts]: uh what do u mean

**Review by nullpixel — APPROVED** [review]
This can be merged when merge conflicts are resolved!

**Pull request closed without merging**